x-pack/legacy/plugins/spaces/server/lib/get_active_space.ts

```diff
@@ -1,24 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License;
- * you may not use this file except in compliance with the Elastic License.
- */
-
-import { Space } from '../../common/model/space';
-import { wrapError } from './errors';
-import { SpacesClient } from './spaces_client';
-import { getSpaceIdFromPath } from '../../common';
-
-export async function getActiveSpace(
-  spacesClient: SpacesClient,
-  requestBasePath: string,
-  serverBasePath: string
-): Promise<Space> {
-  const spaceId = getSpaceIdFromPath(requestBasePath, serverBasePath);
-
-  try {
-    return spacesClient.get(spaceId);
-  } catch (e) {
-    throw wrapError(e);
```

Comment:
__question:__ I see we have only `SpacesService.getActiveSpace`, but we don't wrap error there, do you remember why we were wrapping it here?

Reply:
I sadly don't remember the history. My current thinking for `wrapError` is that it should be used as a way to translate "any error" into a suitable payload for `responseToolkit.customError()`. In other words, I'm only expecting to wrap errors at/near route handlers and request interceptors, where I need to respond in a well-formed fashion to anything that could be thrown.

Given that, maybe `wrapError` would be better off renamed to something like `toCustomResponseError()` or similar? What do you think?

Reply:
> In other words, I'm only expecting to wrap errors at/near route handlers and request interceptors, where I need to respond in a well-formed fashion to anything that could be thrown.

Yeah, I definitely agree!

> Given that, maybe wrapError would be better off renamed to something like toCustomResponseError() or similar? What do you think?

`toCustomResponseError` sounds good to me and perfectly describes what it does, but I think it's also fine if we keep it as is for now. We have the same thing in Security plugin :see_no_evil: and can probably unify and rename those in one go once at some point.

Reply:
I'll leave it as-is for now then